boards/arduino-leonardo/src/lib.rs

```diff
@@ -45,15 +45,20 @@ pub extern crate atmega32u4_hal as hal;
 /// See [`avr_device::entry`](https://docs.rs/avr-device/latest/avr_device/attr.entry.html).
 #[cfg(feature = "rt")]
 pub use hal::entry;
+pub use hal::usart;
 
 mod pins;
 pub use crate::pins::*;
 
-pub use atmega32u4_hal::atmega32u4;
 pub use crate::atmega32u4::Peripherals;
-pub use atmega32u4_hal::prelude;
+pub use atmega32u4_hal::atmega32u4;
 
+pub mod prelude {
+    // Expose everything from the HAL prelude
+    pub use atmega32u4_hal::prelude::*;
 
+    pub use hal::usart::BaudrateArduinoExt as _;
+}
```

Comment:
Please rebase this branch/merge upstream changes into it, as the imports here have slightly changed in `master`.  The prelude module looks good, though!

Reply:
done, not sure if in correct way

Reply:
Looks good to me!